Makefile

```diff
@@ -14,6 +15,9 @@ test:
 $(DRIVER):
 	cd windows-gaming-driver && cargo build --release --locked
 
+$(VFIO_UBIND):
+	cd vfio-ubind && cargo build --release --locked
```

Comment:
So is this postponed now or what? It's just that the major reason why I would prefer doing it with workspaces is that we bridge make to cargo by always invoking cargo which gets weird/slow fast when we have 10 cargo invocations every time.